### PR TITLE
Add KMS quickstart

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: kms/api/quickstart.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+def run_quickstart():
+    # [START kms_quickstart]
+    # Imports the Google APIs client library
+    from googleapiclient import discovery
+    from oauth2client.client import GoogleCredentials
```
Comment (jonparrott) [inline kms/api/quickstart.py]: This is no longer needed, googleapiclient can figure out application default credentials on its own.

File: kms/api/quickstart.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    # Your Google Cloud Platform project ID
+    project_id = 'YOUR_PROJECT_ID'
+
+    # The "global" zone will list all keys, can be a specific zone if desired
+    zone = 'global';
+
+    # Acquires credentials
+    credentials = GoogleCredentials.get_application_default().create_scoped(
```
Comment (jonparrott) [inline kms/api/quickstart.py]: Likewise here.

File: kms/api/quickstart.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        ['https://www.googleapis.com/auth/cloud-platform'])
+
+    # Instantiates a client
+    kms_client = discovery.build('cloudkms', 'v1beta1', credentials=credentials)
+
+    # Lists key rings
+    req = kms_client.projects().locations().keyRings().list(
+        parent='projects/{}/locations/{}'.format(project_id, zone))
```
Comment (jonparrott) [inline kms/api/quickstart.py]: make parent its own variable, please.

File: kms/api/quickstart.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    # Acquires credentials
+    credentials = GoogleCredentials.get_application_default().create_scoped(
+        ['https://www.googleapis.com/auth/cloud-platform'])
+
+    # Instantiates a client
+    kms_client = discovery.build('cloudkms', 'v1beta1', credentials=credentials)
+
+    # Lists key rings
+    req = kms_client.projects().locations().keyRings().list(
```
Comment (jonparrott) [inline kms/api/quickstart.py]: `request` and `response`, we don't get charged by the letter.

File: kms/api/quickstart.py (hunk 1 above)
Comment (jmdobry) [inline kms/api/quickstart.py]: Done.

File: kms/api/quickstart.py (hunk 2 above)
Comment (jmdobry) [inline kms/api/quickstart.py]: Done.

File: kms/api/quickstart.py (hunk 4 above)
Comment (jmdobry) [inline kms/api/quickstart.py]: Done.

File: kms/api/quickstart.py (hunk 3 above)
Comment (jmdobry) [inline kms/api/quickstart.py]: Done.

**Comment by jonparrott** [comment]
Thank you, @jmdobry!

**Pull request merged**